### ISSUE TITLE
javascript-json-client documentation contains server config hints

### ISSUE THREAD

**Issue by vgcookies** [stoicflame/enunciate#410]
Hi.

I took a look at the new `javascript-json-client` generated documentation. It seems that somehow the documentation of the `enunciate.xml` (?) has found its way into it :)?

```
JavaScript Browser Global Configuration Example

<enunciate>
  <modules>
    <javascript-client slug="myslug"/>
```

It's nice, that it tells you how to configure it, but isn't this only important to the server and not the client trying to download and use a client :)? Or I'm missing something?

**Comment by stoicflame** [comment]
Agreed, that shouldn't be in there. I'll get it fixed. Thanks for the report.

**Comment by stoicflame** [comment]
Fixed at 9282113649394a80153da5d1701faf68cec2e085.

**Comment by stoicflame** [comment]
[Enunciate 2.5.0 is now available.](https://github.com/stoicflame/enunciate/releases/tag/v2.5.0)
